app/scripts/modules/ecs/metricAlarm/MetricAlarm.ts

```diff
@@ -0,0 +1,7 @@
+export interface MetricAlarmDescriptor {
```

Comment:
Should be prefixed with `I`, i.e. `IMetricAlarmDescriptor`; file name should match, unless you intend to add some more interfaces in here.